hyrisecockpit/drivers/__default__/default_workload.py

```diff
@@ -0,0 +1,34 @@
+"""Module for default workload."""
+from collections import OrderedDict
+from random import choice, choices
+from typing import List
+
+from hyrisecockpit.drivers.__default__.task_types import DefaultTask
+from hyrisecockpit.drivers.__default__.workload_reader import WorkloadReader
+
+
+class DefaultWorkload:
+    """Generates workloads from queries."""
+
+    def __init__(self, benchmark: str, scalefactor: float, query_path: str):
```

Comment:
```suggestion
    def __init__(self, benchmark: str, scale_factor: float, query_path: str):
```
I noticed its written `scale_factor` in the `app/` files and `scalefactor` here. We should consolidate the naming.